Revamp UI

suibun-code opened the pull request.

suibun-code committed:
Revamp UI

suibun-code committed:
Revamp UI

Downmoto committed:
Database Path fixes

Downmoto commented:
Hey, I won't merge this into main yet as it still needs a little work. The main content loads way too far to the right, It also does that in the current version, but the sidebar there is longer so it doesn't appear as wonky. Also please add the react-icons library you used as a dev dependency, it's not in there currently. One other note, swap from the all-files version to the standard one as the all-files version makes build times longer than needed. Other than that, the switch to manitee is dope, we'll further develop the aesthetics in sprint 3 using manitee.

Downmoto committed:
homepage launch on default, window maxsize and no menubar

Downmoto committed:
correctly links to /

Downmoto committed:
fixed major bug that would disable inputs when goal saved

Downmoto commented:
Your images in Teams seem to be working as intended. this is what it looks like when I run it:
![image](https://github.com/user-attachments/assets/7d9fe02f-3ed7-463c-951b-13c779722af8)

vyatsenk0 committed:
Merge pull request #8 from PRJ666Group1/Arad

Arad

suibun-code committed:
Merge remote-tracking branch 'origin/Roma' into Roma

suibun-code committed:
Fix UI issues in previous commit

suibun-code committed:
Fix '/main_widow' to '/'

Same as Arad's commit. Was not working on my end at first so I changed back to main_window, realized the changes Arad made in his commit, and applied the same here
